tests_integration/snowflake_connector.py

```diff
@@ -6,24 +6,48 @@
 import pytest
 from snowflake import connector
 from snowflake.cli.api.exceptions import EnvironmentVariableNotFoundError
+from contextlib import contextmanager
 
 _ENV_PARAMETER_PREFIX = "SNOWFLAKE_CONNECTIONS_INTEGRATION"
 
 
-@pytest.fixture(scope="function")
-def test_database(snowflake_session):
-    database_name = f"db_{uuid.uuid4().hex}"
+def add_uuid_to_name(name: str) -> str:
+    return f"{name}_{uuid.uuid4().hex}"
+
+
+@contextmanager
+def setup_test_database(snowflake_session, database_name: str):
+    database_name = f'"{database_name}"'
     snowflake_session.execute_string(
         f"create database {database_name}; use database {database_name}; use schema public;"
     )
-    os.environ[f"{_ENV_PARAMETER_PREFIX}_DATABASE"] = database_name
+    os.environ[f"{_ENV_PARAMETER_PREFIX}_DATABASE"] = f"{database_name}"
```

Comment:
Do you make string from string in `""`? 🤔

Reply:
Refactor leftover - fixed :)